tests/011-test-docker-root-volume-cleanup.sh

```diff
@@ -0,0 +1,59 @@
+source $SRCDIR/libtest.sh
+
+# Test "docker-storage-setup --reset" for DOCKER_ROOT_VOLUME=yes.
+# Returns 0 on success and 1 on failure.
+test_reset_docker_root_volume() {
+  local devs=${TEST_DEVS}
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+  local mount_filename="var-lib-docker.mount"
+  local docker_root_lv_name="docker-root-lv"
+
+  # Error out if any pre-existing volume group vg named dss-test-foo
+  if vg_exists "$vg_name"; then
+    echo "ERROR: $testname: Volume group $vg_name already exists." >> $LOGS
+    return $test_status
+  fi
+
+  cat << EOF > /etc/sysconfig/docker-storage-setup
+DEVS="$devs"
+VG=$vg_name
+DOCKER_ROOT_VOLUME=yes
+EOF
+
+ # Run docker-storage-setup
+ $DSSBIN >> $LOGS 2>&1
+
+ # Test failed.
+ if [ $? -ne 0 ]; then
+    echo "ERROR: $testname: $DSSBIN Failed." >> $LOGS
+    cleanup $vg_name "$devs"
+    return $test_status
+ fi
+
+  $DSSBIN --reset >> $LOGS 2>&1
+  # Test failed.
+  if [ $? -eq 0 ]; then
+     test_status=0
+     if [ -e "/etc/sysconfig/docker-storage" ] || [ -e "/etc/systemd/system/${mount_filename}" ]; then
+          test_status=1
+     fi
+     if lv_exists "$vg_name" "$docker_root_lv_name";then
+          test_status=1
+     fi
```

Comment:
I think we can reuse `cleanup_container_root_volume` (Rename that to cleanup_all() ).
https://github.com/projectatomic/docker-storage-setup/blob/master/tests/010-test-use-devs-to-create-docker-root-volume.sh#L57-L67 if we move this to `libtest.sh`.
It does the following:
1) Unmount the mount path.
2) Cleanup `docker-root-lv`.
3) Cleanup the mount file.